.env

```diff
@@ -30,5 +30,9 @@ DATABASE_URL=sqlite:///%kernel.project_dir%/data/database.sqlite
 ###< doctrine/doctrine-bundle ###
 
 ###> symfony/mailer ###
-# MAILER_DSN=smtp://localhost
+MAILER_DSN=smtp://05f5a706f7f5d7:29dabafe16ab0a@smtp.mailtrap.io:2525
```

Comment:
.env file should not contain real credentials as it's committed to git. Store real DSN in .env.local file that is in .gitignore